api/src/swagger/config.yaml

```diff
@@ -58,7 +58,7 @@ paths:
           description: 'Successful query'
         '404':
           description: 'Invalid input'
-  /compartments/{id}?model={model}:
+  /compartments/{id}?model={model}&full=true:
```

Comment:
Wouldn't it make sense to add `full` to the list of parameters (line 71) as well?

Reply:
Not quite sure if we should add this as an additional parameter. And if it is added, the description text should also be changed.

There's another thing which confused me is how does this parameter `full` work. 
It seems both of the following two urls 
`/api/v2/compartments/golgi_apparatus?model=HumanGem&full=false`
`/api/v2/compartments/golgi_apparatus?model=HumanGem&full=true"`
return the same result. 
while 
`/api/v2/compartments/golgi_apparatus?model=HumanGem&full` and 
`/api/v2/compartments/golgi_apparatus?model=HumanGem`
return a different result.
Any ideas from @e0 , @inghylt, @mihai-sysbio and @MalinAhlberg

Reply:
Good catch @MalinAhlberg! I think it makes sense to add it as a parameter in the list of `/compartments/{id}?model={model}` parameters since it is a parameter we provide to get the result specified in the description. Given @nanjiangshu discovery though, I'm not really sure how we would do it since it seems like you need to omit it completely to get change in result, a bit silly instruction IMO. Either we fix the neo4j code in this PR or create a new issue to work with this, not sure what is optimal.